src/components/TextField/TextField.vue

```diff
@@ -5,11 +5,14 @@ Labelled(
   :action="labelAction",
   :labelHidden="labelHidden",
   :requiredIndicator="requiredIndicator",
+  :label="label",
 )
   template(#label, v-if="slots.label")
     slot(name="label")
   template(#help-text, v-if="slots['help-text']")
     slot(name="help-text")
+  template(#help-text, v-else-if="helpText")
```

Comment:
There are many components using `help-text` , so I think we should keep using slot, or unless we have to update all.

Reply:
I'd suggest we update all to allow anything that can be simply text to also allow params. This way you can create any component that only requires text with simple params. If you then need more than just text you can use a slot? Or am I missing something here?

Reply:
No, you're totally correct, but we need to do it consistently, because there are a lot of components have `help-text` slot. 
That means you should update all of them. 

By the way, you need to update the doc too.